tests/Loader/LoaderIntegrationTest.php

```diff
@@ -1978,6 +1978,61 @@ public function provideLegacyFixturesToInstantiate()
             'An error occurred while generating the fixture "dummy" (Nelmio\Alice\Entity\Instantiator\DummyWithNamedPrivateConstructor): Could not instantiate fixture "dummy".',
             GenerationThrowable::class,
         ];
+
+        yield 'with constructor named parameters' => [
```

Comment:
I think this is a little error of a merge. The current cases have been appended to `provideLegacyFixturesToInstantiate()` in which the deprecation message is expected (hence the CI failure). They should instead be moved in `provideFixturesToInstantiate()`.

I also notice i did a mistake with a PR earlier and there is still commented cases in `provideFixturesToInstantiate()`. If you can remove them while you at it it would be amazing

Reply:
Ah thanks for the hint! I overlooked this when rebasing. I removed commented out code too (as a dedicated commit).